snapcraft/meta.py

```diff
@@ -140,16 +140,18 @@ def _copy(meta_dir, relpath, new_relpath=None):
     return os.path.join('meta', os.path.basename(relpath))
 
 
-def _copy_security_profiles(meta_dir, apps):
-    # TODO: remove once capabilities are implemented.
-    for app in apps:
-        if 'security-policy' in apps[app]:
-            apps[app]['security-policy']['apparmor'] = \
-                _copy(meta_dir, apps[app]['security-policy']['apparmor'])
-            apps[app]['security-policy']['seccomp'] = \
-                _copy(meta_dir, apps[app]['security-policy']['seccomp'])
+def _copy_security_profiles(meta_dir, uses):
+    # TODO: remove once skills are implemented.
+    for slot in uses:
+        if uses[slot]['type'] != 'migration-skill':
```

Comment:
if slot name itself is migration-skill then this should work too (type defaults to slot name)